package-res/ccc/core/base/chart/chart.axes.js

```diff
@@ -433,6 +456,30 @@ pvc.BaseChart
             }
         }
 
+        // CDF603
+        var width;
+        if(axis.option.isDefined('FixedLength')) {
+            width = axis.option('FixedLength');
+            if(width < 0) width = -width; // Length is always an absolute value
+        }
+
+        // CDF603
+        // If FixedMin + FixedLength specified, max is directly set
+        // using both and FixedMax is ignored
+        // if width is not null then FixedLength was defined
+        if(min   != null && axis.option.isDefined('FixedMin') && 
+           width != null && max == null) {
+            max = min - (0-width);
```

Comment:
Please use the slightly less intriguing idiom for ensuring Dates are converted to numbers:
```javascript
max = min + (+width);
```
Although the parentheses aren't required they help in informing what's being done.